azure-mgmt/tests/test_batch.py

```diff
@@ -215,7 +215,8 @@ def setUpClass(cls):
             cls.batch_client_sk, cls.batch_client_aad = create_batch_account(cls.batch_mgmt_client, cls.settings, cls.live)
         except Exception as err:
             cls.tearDownClass()
-            raise AssertionError("Failed to setup Batch Account: {}".format(err))
+            raise
```

Comment:
Oops?

IIRC I hit this at some point and the original raise was obscuring the actual exception. I'll revert it.